drake/examples/Pendulum/pendulum_system.h

```diff
@@ -0,0 +1,138 @@
+#pragma once
+
+#include <memory>
+
+#include "drake/drakePendulumSystem_export.h"
+#include "drake/systems/framework/basic_vector.h"
+#include "drake/systems/framework/leaf_system.h"
+
+namespace drake {
+namespace examples {
+namespace pendulum {
+
+/// The state of a one-dimensional pendulum system.
+///
+/// @tparam T The vector element type, which must be a valid Eigen scalar.
+///
+/// Instantiated templates for the following kinds of T's are provided:
+/// - double
+/// - AutoDiffXd
+template <typename T>
+class DRAKEPENDULUMSYSTEM_EXPORT PendulumStateVector
```

Comment:
You can not export a template, you can only export instances of specific types.  Try removing this:  DRAKEPENDULUMSYSTEM_EXPORT.